docs-source/installation.rst

```diff
@@ -1,84 +1,27 @@
 Installation
 ============
 
-.. _installation:
+Installing Vehicle itself
+-------------------------
 
-At the moment, the only way to install Vehicle is from source. We aim to make it
-available as both an executable and a Python package in the near future.
+Installing Vehicle is very easy via the Python package manager.
+Simply run ``pip install vehicle-lang``. This will both add the
+executable ``vehicle`` to the PATH variable allowing you to call
+the compiler directly, as well as making the Python bindings
+importable as you would any other module.
```

Comment:
@wenkokke do we have a list of architectures/platforms that the wheels are known not to work on?